va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -812,34 +884,118 @@ def shift_granularity(current_granularity, levels, move_up=False):
 
 # =========Map Metrics =======================#
 # Top metrics to track for map dropdown
-@app.callback(
+# callback 5
+# @app.callback(
+#     Output(component_id="cod_type", component_property="options"),
+#     [
+#         Input(component_id="va_data", component_property="data"),
+#         Input(component_id="filter_dict", component_property="data"),
+#     ],
+# )
+# def get_metrics(va_data, filter_dict=None, N=10, **kwargs):
+#     # by default, start with aggregate measures
+#     # print(va_data)
+#     START_TIME = time.time()
+#     metrics = []
+#     metric_data = pd.DataFrame(va_data)
+#     if metric_data.size > 0:
+#         if filter_dict is not None:
+#             metric_data = metric_data.loc[filter_dict["ids"]["valid"], :]
+#             # only load options if remaining data after filter
+#             if metric_data.size > 0:
+#                 # add top N CODs by incidence to metric list
+#                 metrics = ["all"] + (
+#                     metric_data["cause"]
+#                     .value_counts()
+#                     .sort_values(ascending=False)
+#                     .head(N)
+#                     .index.tolist()
+#                 )
+#     print("get metrics Callback", time.time() - START_TIME)
+#     return [{"label": LOOKUP["display_names"].get(m, m), "value": m} for m in metrics]
+
+### client-side conversion for get_metrics ###
+app.clientside_callback(
+    """
+    function(va_data, filter_dict, display_names){
```

Comment:
this function is really hard to read, its formatted as a string block but much of it is comments. please delete the commented out code especially if its unimportant